src/gui/mainwindow.cpp

```diff
@@ -1111,6 +1118,9 @@ void MainWindow::showEvent(QShowEvent *e)
         move(Utils::Misc::screenCenter(this));
         m_posInitialized = true;
     }
+#ifdef Q_OS_WIN
+     setupTaskbarButton();
```

Comment:
You can move it under previous condition (`if (!m_posInitialized)`).

Reply:
unfortunately that part doesn't get called if the window geometry is stored in settings, it will get called probably only on first run after installation